test/models/report_test.rb

```diff
@@ -3,7 +3,19 @@
 require 'test_helper'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test '#editable' do
+    alice = users(:alice)
+    bob = users(:bob)
+    alice_report = reports(:alice_report)
+
+    assert alice_report.editable?(alice)
+    assert_not alice_report.editable?(bob)
+  end
```

Comment:
```suggestion
  test '#editable' do
    assert eports(:alice_report).editable?(users(:alice))
    assert_not eports(:alice_report).editable?(users(:bob))
  end
```

一旦一時変数に入れなくても名前が自明なのでこういう感じでもいいかもと思いました。

Reply:
なるほどです！
やってみたところ、とても短くなりました！😄